installer/bundle/bundle_skel_AIX.sh

```diff
@@ -0,0 +1,432 @@
+then
+    echo "Failed: could not extract the install bundle."
+    cleanup_and_exit ${STATUS}
+fi
+
+#
+# Do stuff after extracting the binary here, such as actually installing the package.
+#
+
+EXIT_STATUS=0
+SCX_EXIT_STATUS=0
+OMI_EXIT_STATUS=0
+BUNDLE_EXIT_STATUS=0
+
+case "$installMode" in
+    E)
+        # Files are extracted, so just exit
+        cleanup_and_exit 0 "SAVE"
+        ;;
+
+    I)
+        echo "Installing cross-platform agent ..."
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            check_if_pkg_is_installed omi
+            if [ $? -eq 0 ]; then
+                pkg_upd $OMI_PKG omi
+                # It is acceptable that this fails due to the new omi being 
+                # the same version (or less) than the one currently installed.
+                OMI_EXIT_STATUS=0
+            else
+                pkg_add $OMI_PKG omi
+                OMI_EXIT_STATUS=$?
+            fi
+        fi
+
+        pkg_add $OM_PKG scx
+        SCX_EXIT_STATUS=$?
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            # Install bundled providers
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            for i in *-oss-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+                ./$i
+                if [ $? -eq 0 ]; then
+                    OSS_BUNDLE=`basename $i -oss-test.sh`
+                    ./${OSS_BUNDLE}-cimprov-*.sh --install $FORCE $restartDependencies
+                    TEMP_STATUS=$?
+                    [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
+                fi
+            done
+        fi
+        ;;
+
+    U)
+        echo "Updating cross-platform agent ..."
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            check_if_pkg_is_installed omi
+            if [ $? -eq 0 ]; then
+                pkg_upd $OMI_PKG omi $?
+                # It is acceptable that this fails due to the new omi being 
+                # the same version (or less) than the one currently installed.
+                OMI_EXIT_STATUS=0
+            else
+                pkg_add $OMI_PKG omi
+                OMI_EXIT_STATUS=$?  
+            fi
+        fi
+
+        pkg_upd $OM_PKG scx $?
+        SCX_EXIT_STATUS=$?
+
+        if [ $PROVIDER_ONLY -eq 0 ]; then
+            # Upgrade bundled providers
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            echo "----- Updating bundled packages -----"
+            for i in *-oss-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+                ./$i
+                if [ $? -eq 0 ]; then
+                    OSS_BUNDLE=`basename $i -oss-test.sh`
+                    ./${OSS_BUNDLE}-cimprov-*.sh --upgrade $FORCE $restartDependencies
+                    TEMP_STATUS=$?
+                    [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
+                fi
+            done
+        fi
```

Comment:
No mega-bundles for UNIX, lines 400-416 can go away here (and similar code in all other UNIX-based files).